src/Symfony/Bridge/Doctrine/SchemaListener/DoctrineDbalCacheAdapterSchemaSubscriber.php

```diff
@@ -11,16 +11,20 @@
 
 namespace Symfony\Bridge\Doctrine\SchemaListener;
 
+use Doctrine\Common\EventSubscriber;
 use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+use Doctrine\ORM\Tools\ToolEvents;
 use Symfony\Component\Cache\Adapter\DoctrineDbalAdapter;
 
 /**
  * Automatically adds the cache table needed for the DoctrineDbalAdapter of
  * the Cache component.
  *
  * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @deprecated since Symfony 6.3
```

Comment:
`, use DoctrineDbalCacheAdapterSchemaListener instead.`
& let's trigger a deprecation before the class declaration
(same in the other deprecated classes)